### DISCUSSION TITLE
When will you release Gradle Play Publisher 3.4.0-agp7.0?

### DISCUSSION THREAD

**Discussion by nuhkoca** [Triple-T/gradle-play-publisher#953] [General]
Hello,

May I know what is your plan to release `Gradle Play Publisher 3.4.0-agp7.0`? We'd like to upgrade `Gradle` to `7.0` in our project and do require this plugin to be released, as well. Thanks!

**Comment by SUPERCILEX** [comment]
It is released, that's the version name. You can see installation instructions for the 7.0 branch here: https://github.com/Triple-T/gradle-play-publisher/tree/7.0#installation

**Comment by nuhkoca** [comment]
Thanks @SUPERCILEX when I saw pre-release label I thought it hadn't been released yet.
